### PR TITLE
Bug 1134868 - Fix intermittent app priming issue

### PR DESCRIPTION
**Pull request by rwood-moz**

### CODE REVIEW

File: lib/suite/cold-launch.js [hunk 1]
```diff
@@ -239,7 +236,13 @@ ColdLaunch.prototype.prime = function() {
 
   // Wait for 6 seconds to give time for pre-allocated process
   setTimeout(function() {
-    runner.launch();
+    return runner.swipeHack()
```
Comment (eliperelman) [inline lib/suite/cold-launch.js]: Move `.swipeHack()` to next line and reindent. Move `wait_after_hack` to runner property in ColdLaunch constructor.

```js
ColdLaunch = function(options) {
  Phase.call(this, options);

  this.swipeHackDelay = this.options.emulator ? 6 * 1000 : 3 * 1000;
};
```

```js

return runner
  .swipeHack()
  .then(function() {
    setTimeout(function() {
      runner.launch();
    }, runner.swipeHackDelay);
  });
```

Comment (rwood-moz) [inline lib/suite/cold-launch.js]: Ok yeah that's cleaner code, thanks

File: lib/suite/cold-launch.js [hunk 2]
```diff
@@ -239,7 +237,13 @@ ColdLaunch.prototype.prime = function() {
 
   // Wait for 6 seconds to give time for pre-allocated process
   setTimeout(function() {
-    runner.launch();
+    return runner
+    .swipeHack()
```
Comment (eliperelman) [inline lib/suite/cold-launch.js]: Indent lines 241-246 2 spaces to the right of the `return` statement above them.

File: lib/suite/cold-launch.js [hunk 3]
```diff
@@ -239,7 +237,13 @@ ColdLaunch.prototype.prime = function() {
...
   // Wait for 6 seconds to give time for pre-allocated process
   setTimeout(function() {
-    runner.launch();
+    return runner
+      .swipeHack()
+      .then(function() {
+        setTimeout(function() {
+          runner.launch();
+        }, runner.swipeHackDelay);
+      })
```
Comment (eliperelman) [inline lib/suite/cold-launch.js]: Missing semicolon.

**Pull request merged**